general/forall.hpp

```diff
@@ -459,7 +459,16 @@ template <typename HBODY>
 void RajaSeqWrap(const int N, HBODY &&h_body)
 {
 #ifdef MFEM_USE_RAJA
-   RAJA::forall<RAJA::loop_exec>(RAJA::RangeSegment(0,N), h_body);
+
+#if (RAJA_VERSION_MAJOR <= 2023 && RAJA_VERSION_MINOR <= 6 && RAJA_VERSION_PATCHLEVEL < 1)
```

Comment:
⚠️  This check will fail for a version like 2022.10.x which is not what you want.

Reply:
Good point, we can just check if we are using version 2023 and above; since that we we need to start using seq_exec.